src/pages/Map/MapPage.tsx

```diff
@@ -1,26 +1,31 @@
+import Weather from 'api/services/Weather';
+import { useLocalStorage } from 'hooks/useLocalStorage';
 import { TileLayer } from 'react-leaflet';
 import { PlaceMarkers } from './components/PlaceMarkers';
 import { StyledMapContainer, MapWrapper } from './MapPage.styled';
 import { MapPageTestIds } from './MapPageTestIds';
 
-const initialBounds = '(50.72167742756552,16.311950683593754,51.48993452350156,17.76351928710938)';
 const initialZoom = 10;
 
 const MapPage = () => {
-  const localization: [number, number] = [51.107, 17.038];
+  const [localValue] = useLocalStorage('current_location');
+  const { data } = Weather.useLocation(localValue || 'Wroclaw');
+  const location = data?.location;
+
+  const defaultPosition: [number, number] = [51.107, 17.038];
 
   return (
     <MapWrapper data-testid={MapPageTestIds.MapPage}>
       <StyledMapContainer
-        center={localization}
-        zoom={10}
+        center={location ? [location.lat, location.lon] : defaultPosition}
```

Comment:
**question** it looks like the map will jump to another position after data is loaded, am i right?